src/selectors/Entities/vaccinePrescription.js

```diff
@@ -50,7 +50,7 @@ export const selectSortedVaccines = createSelector([selectVaccines], vaccines =>
 });
 
 export const selectSelectedRows = createSelector([selectSelectedVaccines], vaccines =>
-  vaccines.reduce((acc, vaccine) => ({ ...acc, [vaccine.id]: true }), {})
+  vaccines.reduce((acc, vaccine) => ({ ...acc, [vaccine?.id]: true }), {})
```

Comment:
🤔  - I'm not sure why there would be null entries in the array. will take your word for it